src/coding-guidelines/expressions.rst

```diff
@@ -17,16 +17,16 @@ Expressions
     :scope: system
     :tags: security, performance, numerics
 
-    To avoid runtime panics and unexpected wraparound behavior, it is important to avoid arithmetic overflow.
-    In cases where wraparound behavior is intentional, explicit wrapping functions can be used.
-    On the other hand, it is important to allow inline arithmetic expressions for readability.
+    Eliminate arithmetic overflow to avoid runtime panics and unexpected wraparound behavior.
+    Use explicit wrapping or saturation semantics in cases where these behaviors are intentional.
     Range checking can be accomplished using a variety of mechanisms, provided that the possibility of arithmetic overflow is elimianted.
+    Inline arithmetic expressions are often important for readability.
```

Comment:
```suggestion
```
This is correct, but it goes somewhere else. Perhaps in an Appendix to the Guidelines? Since it's not a guideline per se.

Anyway. The first bit of a Guideline is the Amplification section. This section is normative, and so every sentence in it is expected to be an actionable requirement of some sort.